### DISCUSSION TITLE
Where are the 1.8.0 artefacts?

### DISCUSSION THREAD

**Discussion by Sciss** [apache/incubator-mxnet#20629] [Q&A]
Hello. Despite the website seemingly referring to outdated code with respect to Scala, I was shown that Scala examples exist within the dedicated 1.8.0 branch. But I don't understand why there are no artefacts for this version. If I understand correctly, 1.8.0 was the last stable version released a year and a half ago, but the maven repositories merely contain 1.7.0-SNAPSHOT and then jump to 2.0.0-SNAPSHOT:

https://repository.apache.org/content/groups/snapshots/org/apache/mxnet/mxnet-full_2.11-linux-x86_64-cpu/
https://repository.apache.org/content/groups/public/org/apache/mxnet/mxnet-full_2.11-linux-x86_64-cpu/

What is the concept of this? Am I supposed to build the project from source? Seems odd to me. Or am I missing a releases repository?

**Comment by szha** [comment]
The artifacts of 1.8 was removed due to licensing issue of some components packaged. For now the path is to build from source until the next stable version is released.
